docs/source/array-design.rst

```diff
@@ -102,32 +103,32 @@ Some ways in which ``chunks`` reflects properties of our array
 Create an Array Object
 ----------------------
 
-So to create an ``da.Array`` object we need a dictionary with these special
-keys ::
+In order to create an ``da.Array`` object we need a dictionary with these special
+keys::
 
     dsk = {('x', 0, 0): ...}
 
-a name specifying to which keys this array refers ::
+a name specifying which keys this array refers to::
 
     name = 'x'
 
 and a chunks tuple::
 
     chunks = ((5, 5, 5, 5), (8, 8, 8))
 
-Then one can construct an array::
+Then, using this elements, one can construct an array::
 
     x = da.Array(dsk, name, chunks)
 
-So ``dask.array`` operations update dask graphs, update dtypes, and track chunk
+In short, ``dask.array`` operations update Dask graphs, update dtypes, and track chunk
 shapes.
 
 
 Example - ``eye`` function
 --------------------------
 
-As an example lets build the ``np.eye`` function for ``dask.array`` to make the
-identity matrix
+As an example, lets build the ``np.eye`` function for ``dask.array`` to make the
+identity matrix:
```

Comment:
Should this be 2 `:`s?

Reply:
It is not necessary in this case because the sample code is inside a block